test/numa/run_tests.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+TESTS_DIR="$1"
+
+usage() {
+    echo "Usage: run_tests.sh TESTS_DIR"
+    echo "TESTS_DIR is expected to be structured as POLICY/TOPOLOGY/TEST with files:"
+    echo "POLICY/cri-resmgr.cfg: configuration of cri-resmgr"
+    echo "POLICY/TOPOLOGY/topology.var.json: contents of the topology variable for run.sh"
+    echo "POLICY/TOPOLOGY/code.var.sh: contents of the code var (that is, test script)"
```

Comment:
This is a good start. But let's try to write a bunch of test cases (somehow) split into a number
of logical groups and see if this fits the bill.